Game/Scripts/UITrigger.cpp

```diff
@@ -75,31 +76,39 @@ void UITrigger::Start()
 
 void UITrigger::Update(float deltaTime)
 {
-
-}
-
-void UITrigger::OnCollisionEnter(ComponentRigidBody* other)
-{
-	if (other->GetOwner()->GetComponent(ComponentType::PLAYER))
+	if(onTriggerState != false)
```

Comment:
onTriggerState is a boolean?
Why aren't we just checking `if(onTriggerState)` or `ìf(!onTriggerState)`? It should do the same if the field is a boolean and it is more readable.

This also applies to other conditions of this file where we do `if(isWhateverTrigger == true)` , that can be refactored to `if(isWhateverTrigger)`

```suggestion
	if (onTriggerState)
```

Reply:
Done!